test/e2e/framework/util.go

```diff
@@ -3394,8 +3394,13 @@ func IssueSSHCommandWithResult(cmd, provider string, node *v1.Node) (*SSHResult,
 	LogSSHResult(result)
 
 	if result.Code != 0 || err != nil {
-		return nil, fmt.Errorf("failed running %q: %v (exit code %d)",
```

Comment:
just add `(exit code %d, stderr %v)` would work.

Reply:
Nice idea, thanks.
I will update this.